src/components/admin/MenuBar.vue

```diff
@@ -1,19 +1,48 @@
 <script setup lang="ts">
+  import { useAuthStore, type AuthStore } from '@/stores/AuthStore';
+  import {
+    usePersonenkontextStore,
+    type PersonenkontextStore,
+    type SystemrechtResponse,
+  } from '@/stores/PersonenKontextStore';
   import { type Ref, ref, type ComputedRef } from 'vue';
   import { onMounted } from 'vue';
   import { useDisplay } from 'vuetify';
 
   const menuDrawer: Ref<boolean> = ref(true);
   const { mobile }: { mobile: ComputedRef<boolean> } = useDisplay();
 
+  const auth: AuthStore = useAuthStore();
+  const personenkontextStore: PersonenkontextStore = usePersonenkontextStore();
+
+  const hasRollenverwaltungRecht: Ref<boolean> = ref(false);
+
   function closeMenuOnMobile(): void {
     if (mobile.value) {
       menuDrawer.value = false;
     }
   }
 
-  onMounted(() => {
-    menuDrawer.value = !mobile.value;
+  // Retrieves logged in user info on mounted and check using the user's sub if he has the right "ROLLEN_VERWALTEN". If not (error from the endpoint) then
```

Comment:
Wird auf die sub oder die personId getestet?

Reply:
War sub und ist jetzt pesronId! comment is veraltet.